### PR TITLE
Account CSV parser

### PR DESCRIPTION
**Pull request by aaronc**
Parses an `accounts.csv` file with four fields:
* address
* balance
* start date
* number of distributions

### CODE REVIEW

File: parse.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+type Record struct {
+	Address                 sdk.AccAddress
+	TotalAmount             apd.Decimal
+	StartTime               time.Time
+	NumMonthlyDistributions int
+}
+
+const (
+	SecondsPerYear  = 31556952
```
Comment (clevinson) [inline parse.go:27]: Shouldn't this be 31536000? (60*60*24*365)

Or 31557600 if we want to account for leap year? (`(60*60*24*365)*(3/4)+(60*60*24*366)*(1/4)`)

File: parse.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+		}
+
+		records = append(records, record)
+	}
+
+	return records, nil
+}
+
+func parseLine(line []string, genesisTime time.Time) (Record, error) {
+	addr, err := sdk.GetFromBech32(line[0], "regen")
```
Comment (clevinson) [inline parse.go]: Isn't hte point of instantiating an SDK config in init.go so that we don't have to hardcode the prefix here?

**Review by clevinson — DISMISSED** [review]
lgtm.

Small q on SecondsPerYear calculation

File: parse.go (hunk 2 above)
Comment (clevinson) [reply parse.go]: OK i guess we need SDK config elsewhere (later down the line), and here maybe it's easier to just hardcode than pulling in a config...

File: parse.go (hunk 1 above)
Comment (aaronc) [reply parse.go:27]: https://www.rapidtables.com/calc/time/seconds-in-year.html

Using 365.2425 days/year

File: parse.go (hunk 2 above)
Comment (aaronc) [reply parse.go]: I should use a const at least

File: parse.go [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+type Record struct {
+	Address                 sdk.AccAddress
+	TotalAmount             apd.Decimal
+	StartTime               time.Time
+	NumMonthlyDistributions int
+}
+
+const (
+	SecondsPerYear  = 31556952
```
Comment (clevinson) [inline parse.go:27]:
```suggestion
	// Based on 365.2425 days/year
	SecondsPerYear  = 31556952
```

**Review by clevinson — DISMISSED** [review]
Suggesting to add the comment in about days/year. Otherwise lgtm

**Review by clevinson — APPROVED** [review]
ack

**Pull request merged**